flow.yml

```diff
@@ -15,10 +15,9 @@ executors:
     replicas: 1  # change this if you have larger VRAM
   - name: clip_encoder
     uses: jinahub+docker://CLIPTorchEncoder/latest-gpu
-    host: 'demo-cas.jina.ai'
+    host: 'api.clip.jina.ai'
     port: 2096
-    tls: true
-    external: true
+    external: false
```

Comment:
why is not this true?

Reply:
The clip-as-service is protected by authentication. It needs a token to be provided ahead of.

Reply:
From the UX perspective, I would suggest using an external executor as an option.

Reply:
okey